tests/monitoring/monitoring_service/test_cli.py

```diff
@@ -11,35 +12,59 @@
     MonitoringService=DEFAULT,
     HTTPProvider=DEFAULT,
 )
+
 DEFAULT_ARGS: List[str] = [
-    '--private-key', '0x' + '1' * 40,
+    '--password', KEYSTORE_PASSWORD,
     '--state-db', ':memory:',
 ]
 
 
-def test_success():
-    """ Calling the request_collector with default args should succeed after heavy mocking """
+def test_success(keystore_file):
+    """ Calling the monitoring_service with default args should succeed after heavy mocking """
     runner = CliRunner()
     with patch.multiple(**patch_args):
-        result = runner.invoke(main, DEFAULT_ARGS, catch_exceptions=False)
+        result = runner.invoke(
+            main,
+            DEFAULT_ARGS + ['--keystore-file', keystore_file],
```

Comment:
`'--keystore-file', keystore_file` could be added to the default args if we made the default args a fixture. Not a big improvement right now, but at least if we get more complicated default args we should do it instead of repeating more args across the tests.

Reply:
True that! Fixed.